index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 id="title"> freeCodeCamp Survey form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="" id="survey-form">
+      <div class="div-form">
+        <label for="name" id="name-label">Name</label>
+        <input type="text" id="name" placeholder="Enter the name" required />
+      </div>
+      <div class="div-form">
+        <label for="email" id="email-label">Email</label>
+        <input
+          type="email"
+          id="email"
+          placeholder="Enter the email address"
+          required
+        />
+      </div>
+      <div class="div-form">
+        <label for="number" id="number-label">Age</label>
+        <input
+          type="number"
+          id="number"
+          min="18"
+          max="100"
+          placeholder="Enter your age"
+          required
+        />
+      </div>
+      <div class="div-form">
+        <label for="role" id="role-label"
+          >Which option best describes your current role?</label>
```

Comment:
# Changes Required ♻️ :
Please make changes to the value of the attribute "for", the value of the “for” attribute of the “label” element should be the ID of a non-hidden form control. This is a more professional way of writing a label tag.
Thanks 😁

Reply:
- [ ] Kindly remove changes required here